packages/integration-tests/api/ems.js

```diff
@@ -1,45 +1,17 @@
 'use strict';
 
-const { lambda } = require('@cumulus/aws-client/services');
-const { callCumulusApi } = require('./api');
+const emsApi = require('@cumulus/api-client/ems');
+const { deprecate } = require('@cumulus/common/util');
 
-/**
- * Fetch deployment's `ems_*` environment variables.
- *
- * @param {string} lambdaName - deployment prefix
- * @returns {Promise<Object>} map of ems_* lambda envs
- */
 async function getLambdaEmsSettings(lambdaName) {
-  const config = await lambda().getFunctionConfiguration({ FunctionName: lambdaName }).promise();
-  const envs = config.Environment.Variables;
-  return Object.keys(envs).reduce((map, key) => {
-    if (!key.startsWith('ems_')) return map;
-    const shortKey = key.slice('ems_'.length);
-    return { ...map, [shortKey]: envs[key] };
-  }, {});
+  deprecate('@cumulus/integration-tests/ems.getLambdaEmsSettings', '1.20.0', '@cumulus/cumulus-api-client/ems.getLambdaEmsSettings');
+  return emsApi.getLambdaEmsSettings(lambdaName);
 }
 
-/**
- * Post a request to the ems API
- *
- * @param {Object} params - params
- * @param {string} params.prefix - the prefix configured for the stack
- * @param {Object} params.request - request body to post
- * @returns {Promise<Object>} - promise that resolves to the output of the API lambda
- */
-async function createEmsReports({ prefix, request }) {
-  return callCumulusApi({
-    prefix: prefix,
-    payload: {
-      httpMethod: 'POST',
-      resource: '/{proxy+}',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      path: '/ems',
-      body: JSON.stringify(request)
-    }
-  });
+
+async function createEmsReports(params) {
+  deprecate('@cumulus/integration-tests/ems.getLambdaEmsSettings', '1.20.0', '@cumulus/cumulus-api-client/ems.getLambdaEmsSettings');
```

Comment:
The version on these deprecation notices needs to be updated to 1.21.0. Sorry, I missed that before

Reply:
Also, there is an easy way to do this via find & replace in vscode if you want to chat offline

Reply:
No worries, I'd already seen that and will update.

Reply:
8bfc97e01

Reply:
and 084716b4f